x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/tabs/index.test.tsx

```diff
@@ -69,8 +69,7 @@ describe('Timeline', () => {
       expect(screen.getByTestId(esqlTabSubj)).toBeVisible();
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/194510
-    describe.skip('no existing esql query is present', () => {
+    describe('no existing esql query is present', () => {
```

Comment:
I see that this test was still failing 5 days ago on a `8.16` branch. Do you feel we've made enough changes that it wouldn't be flaky anymore in `main`